04_health/neonatal_health_code_2022.do

```diff
@@ -0,0 +1,701 @@
+save "${health_data}lbw_births_by_county_`sub'_`y2'.dta", replace
+}
+
+
+//# Nonmissing birth weight births - By Race
+foreach sub of global sub {
+	import delimited using "${health_data}nomiss_bw_by_county_`sub'_`y2'.txt", clear
+		keep countyofresidence countyofresidencecode births	// do not need CDC notes	
+			
+			rename countyofresidence county_name
+			rename countyofresidencecode fips
+			rename births nomiss_births 					// count of births with nonmissing birth weight data
+	
+	destring nomiss_births, replace force
+	
+	gen subgroup_type = "race-ethnicity"	
+	
+	sort fips county_name
+	keep if !missing(fips)								// only keeping observations with data
+
+save "${health_data}nomiss_bw_by_county_`sub'_`y2'.dta", replace
+}
+
+//# open and clean data: mother's education
+//# Low Birthweight Births - By Mother's Education
+
+foreach ed of global ed_sub {
+	import delimited using "${health_data}lbw_births_by_county_`ed'_`y2'.txt", clear
+	di "`ed'"
+		codebook births
+		keep countyofresidence countyofresidencecode births	// do not need CDC notes
+		
+		cap drop if births=="Missing County"					// only keeping observations with birth data	
+			
+			rename countyofresidence county_name
+			rename countyofresidencecode fips
+			rename births lbw_births
+			
+	destring lbw_births, replace force
+	
+	gen subgroup_type = "mothers-education"	
+	
+	sort fips county_name								// count of low birthweight births
+	keep if !missing(fips)								// only keeping observations with data
+save "${health_data}lbw_births_by_county_`ed'_`y2'.dta", replace
+}
+
+//# Nonmissing birth weight births - By Mother's Education
+
+foreach ed of global ed_sub {
+	import delimited using "${health_data}nomiss_bw_by_county_`ed'_`y2'.txt", clear
+		keep countyofresidence countyofresidencecode births	// do not need CDC notes	
+			
+			rename countyofresidence county_name
+			rename countyofresidencecode fips
+			rename births nomiss_births 					// count of births with nonmissing birth weight data
+	
+	destring nomiss_births, replace force
+	
+	gen subgroup_type = "mothers-education"	
+	
+	sort fips county_name
+	keep if !missing(fips)								// only keeping observations with data
+
+save "${health_data}nomiss_bw_by_county_`ed'_`y2'.dta", replace
+}
+
+//# Merge Files
+//# merge files: all births
+use "${health_data}lbw_births_by_county_`y2'.dta", clear
+	merge 1:1 fips county_name using "${health_data}nomiss_bw_by_county_`y2'.dta"
+		tab _merge
+		drop _merge
+save "${health_data}neonatal_health_intermediate_all_`y2'.dta", replace
+
+//# merge files: race/ethnicity
+foreach sub of global sub {
+use "${health_data}lbw_births_by_county_`sub'_`y2'.dta", clear
+	merge 1:1 fips county_name using "${health_data}nomiss_bw_by_county_`sub'_`y2'.dta"
+		tab _merge
+		drop _merge
+		gen `sub' = 1
+	save "${health_data}neonatal_health_intermediate_`sub'_`y2'.dta", replace
+}
+
+//# merge files: mother's education
+foreach ed of global ed_sub {
+	di "`ed'"
```

Comment:
Either delete this line or be consistent in including it in the other loops similar to this one

Reply:
Added